components/scream/src/physics/shoc/atmosphere_macrophysics.cpp

```diff
@@ -49,51 +49,46 @@ void SHOCMacrophysics::set_grids(const std::shared_ptr<const GridsManager> grids
   // Define fields needed in SHOC.
   // Note: shoc_main is organized by a set of 5 structures, variables below are organized
   //       using the same approach to make it easier to follow.
+  std::set<ci_string> q_names =
```

Comment:
Is `q_names` actually used anywhere now?

Reply:
ah, shoot, I thought I deleted it. I copied and pasted it from the `register_fields` method, so I could see which ones needed the "tracers" group, but then forgot to delete the line. Good catch!